catisol.js.org

isolcat opened the pull request:
I want to get a catisol.js.org for my js project, here's the repo [isolcat/CatIsol-UIi](https://github.com/isolcat/CatIsol-UI)

 There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
 I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

indus commented:
#7793 ok I see. The name made me belive it is your personal blog... but this looks different.
Nevertheless the record you have entered still confuses me:

`"isolcat": "catisol-ui.netlify.app"`
would make `isolcat.js.org` point to `catisol-ui.netlify.app`

but I think you want
`"catisol": "catisol-ui.netlify.app"`
that makes `catisol.js.org` point to `catisol-ui.netlify.app`

Because this a not a GitHub page you also don't need [this CNAME](https://github.com/isolcat/CatIsol-UI/blob/master/CNAME) instead you will have to configure Netlify to accept the custom domain.

isolcat commented:
> [#7793 ](https://github.com/js-org/js.org/pull/7793)好的，我明白了。这个名字让我觉得这是你的个人博客......但这看起来不一样。尽管如此，你输入的记录仍然让我感到困惑：
> 
> `"isolcat": "catisol-ui.netlify.app"` would make `isolcat.js.org` point to `catisol-ui.netlify.app`
> 
> 但我认为你想要 `"catisol": "catisol-ui.netlify.app"`这使得`catisol.js.org`指向`catisol-ui.netlify.app`
> 
> 因为这不是GitHub页面，所以您也不需要[此CNAME](https://github.com/isolcat/CatIsol-UI/blob/master/CNAME)，相反，您必须配置Netlify以接受自定义域。



> #7793 ok I see. The name made me belive it is your personal blog... but this looks different. Nevertheless the record you have entered still confuses me:
> 
> `"isolcat": "catisol-ui.netlify.app"` would make `isolcat.js.org` point to `catisol-ui.netlify.app`
> 
> but I think you want `"catisol": "catisol-ui.netlify.app"` that makes `catisol.js.org` point to `catisol-ui.netlify.app`
> 
> Because this a not a GitHub page you also don't need [this CNAME](https://github.com/isolcat/CatIsol-UI/blob/master/CNAME) instead you will have to configure Netlify to accept the custom domain.

Oh, my God, please forgive me. I don't know enough about the mechanism of js.org
The correct way to write it should be what you said: "catisol": "catisol-ui.netlify.app"

indus commented:
I've fixed the alphabetical order for you and I've already added your requested subdomain to JS.ORGs zonefile. You may have to configure your host to accept the custom domain. FYI: https://docs.netlify.com/domains-https/custom-domains/#assign-a-domain-to-a-site
I'll merge after we have "contact". Please keep this PR open.

isolcat commented:
> I've fixed the alphabetical order for you and I've already added your requested subdomain to JS.ORGs zonefile. You may have to configure your host to accept the custom domain. FYI: https://docs.netlify.com/domains-https/custom-domains/#assign-a-domain-to-a-site I'll merge after we have "contact". Please keep this PR open.

I would like to express my gratitude to you from the bottom of my heart, the [domain name](https://catisol.js.org) is now accessible normally, thank you for your help!

indus commented:
Welcome https://catisol.js.org
